integration/dataproc/alluxio.sh

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+#
+#    Copyright 2019 Google, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+set -eux
+
+####################
+# Global constants #
+####################
+readonly MASTER_FQDN="$(/usr/share/google/get_metadata_value attributes/dataproc-master)"
+readonly ROLE="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
+readonly ALLUXIO_DOWNLOAD_PATH="$(/usr/share/google/get_metadata_value attributes/alluxio_download_path || true)"
+readonly ALLUXIO_LICENSE_BASE64="$(/usr/share/google/get_metadata_value attributes/alluxio_license_base64 || true)"
+readonly SPARK_HOME="${SPARK_HOME:-"/usr/lib/spark"}"
+readonly HIVE_HOME="${HIVE_HOME:-"/usr/lib/hive"}"
+readonly HADOOP_HOME="${HADOOP_HOME:-"/usr/lib/hadoop"}"
+readonly PRESTO_HOME="${PRESTO_HOME:-"/opt/presto-server"}"
+readonly ALLUXIO_VERSION="2.3.0-SNAPSHOT"
```

Comment:
This needs to be added to the `./dev/scripts/update-versions.sh` so that it is automatically bumped for each release.

Reply:
got it, updated